### ISSUE TITLE
sql: obj_description produces invalid query

### ISSUE THREAD

**Issue by andymans** [cockroachdb/cockroach#37266]
Just a heads -up. Postico cannot display cockroach tables since the move to 2.19.x.x. 
It displays them fine in 2.1.6.
The error provided by Postico when trying to open a cdb table from the left-hand-side panel is:
"Failed to load table schema."
"Query failed"
"PostgreSQL said: obj_description(): pg_get_objdesc: database "pg_class" does not exist"

Although mildly irritating, this does not prevent a Postico user from viewing a cdb table by issuing the requisite SQL command in the query panel. However it's obviously handy to be able to view tables with a simple button-click.

**Comment by jordanlewis** [comment]
Thanks @andymans for the report. From an initial investigation I can reproduce your problem, looks like something went wrong in our work to add database object comments. We'll make a fix and backport - do appreciate the report.

**Comment by andymans** [comment]
np. will leave the issue open so you can track.

**Comment by awoods187** [comment]
@mjibson this could be a good consideration for expanding the dockerized testing tool you made to help us not break support in new releases

**Comment by jordanlewis** [comment]
Performing automated testing of Postico would be extremely difficult as it is a GUI tool.

**Comment by andymans** [comment]
Yes. Postico is not an 'electron' app. If it were, then potentially such testing could be automated. As it stands, the compatibility with cdb is generally excellent. 
Incidentally, while on the subj of "GUI clients", when using Navicat Premium (includes Postgres support) against cdb 2.19.x.x, the tables can be viewed but with a strange warning dialog when dbl-clicking a table to view it - " ERROR:  column "cl.udt_name" does not exist".  
I haven't seen this issue previously, and obvs do  not have any column called 'cl.udt_name' on any of my tables. Navicat's a popular client (with nice drag/drop query features that ppl like) so it may be worth checking into.

**Comment by jordanlewis** [comment]
@andymans thanks for the other report on Navicat - will send a patch for that as well.

**Comment by jordanlewis** [comment]
@andymans the next 19.1 patch release will have fixes for both reported problems

**Comment by lukeed** [comment]
Problem persists in 19.1.0

```
Build Tag:    v19.1.0
Build Time:   2019/04/29 18:31:15
Distribution: CCL
Platform:     darwin amd64 (x86_64-apple-darwin14)
Go Version:   go1.11.6
C Compiler:   4.2.1 Compatible Clang 3.8.0 (tags/RELEASE_380/final)
Build SHA-1:  25dd36f0139bf65b80758deeeccf35ee17ebd622
Build Type:   release
```

**Comment by jordanlewis** [comment]
Yes, it'll be fixed in 19.1.1. Sorry about that! Please look out for the release in the next week or so.

**Comment by lukeed** [comment]
I skipped over the word "patch" – sorry. Thanks!
